WorkloadManagementSystem/PilotAgent/pilotTools.py

```diff
@@ -39,12 +39,12 @@ def pythonPathCheck():
         if os.path.normpath( p ) in sys.path:
           # In case a given directory is twice in PYTHONPATH it has to removed only once
           sys.path.remove( os.path.normpath( p ) )
-      except Exception, x:
+      except Exception as x:
```

Comment:
As far as I understand this file is also used during pilot installation and should use the old syntax